Templates/GitlabCIPipeline/.gitlab-ci.yml

```diff
@@ -273,33 +335,46 @@ Packaging:
     script: |
         type zowe
         zowe rse issue unix ". ~/.bash_profile && wazideploy-evidence.sh -w ${uniqueWorkspaceId} -l deploy/evidences/evidence.yaml -o deploy/deployment-report.html" --user ${RSEAPI_USER} --password ${RSEAPI_PASSWORD} --cwd ${RSEAPI_WORKING_DIRECTORY}
-        if [ ! -d "${deployOutputDirectory}/" ]; then
-            mkdir ${deployOutputDirectory}
-        fi
+        RC=$?
+        if [ ${RC} -eq 0 ]; then
+            echo [INFO] Successfully generate deployment evidence file.
+        
+            if [ ! -d "${deployOutputDirectory}/" ]; then
+                mkdir ${deployOutputDirectory}
+            fi
 
-        if [ -d "${integrationReportDir}/" ]; then
-            rm -R ${integrationReportDir}
-            echo [INFO] Removed existing directory ${integrationReportDir}
-        fi
-        mkdir ${integrationReportDir}
-        echo [INFO] Create integration report directory = ${integrationReportDir}
+            if [ -d "${integrationReportDir}/" ]; then
+                rm -R ${integrationReportDir}
+                echo [INFO] Removed existing directory ${integrationReportDir}
+            fi
         
-        # Check if Deployment report is generated
-        if [ ! -z "${sourceOutputFile}" ]; then
+            # Create integration report directory
+            mkdir ${integrationReportDir}
+            echo [INFO] Create integration report directory = ${integrationReportDir}
+            # Transfer deployment report and evidence report
             echo [INFO] TRANSFER OUTPUT FILES = ${sourceOutputFile} to ${integrationReportDir}/deployment-report.html
```

Comment:
This all message all in upper case output stands out. Maybe we can be updated to something like this:
`echo [INFO] Loading deployment report ${sourceOutputFile} to ${integrationReportDir}/deployment-report.html`